script/validate/vendor

```diff
@@ -3,7 +3,10 @@
 set -eu -o pipefail
 
 rm -rf vendor/
-vndr |& grep -v -i clone
+# Prevent the data directory from being pruned; this directory does not contain
+# go source files (causing vndr to prune it), however these files are used by
+# golang.org/x/net/publicsuffix through go:embed.
+vndr -whitelist golang.org/x/net/publicsuffix/data |& grep -v -i clone
```

Comment:
Looks like this works; add a `-whitelist` option to prevent `vndr` from pruning the files; we don't have a `make vendor` target, so when we update dependencies, we'll have to remember setting this option.

Reply:
I can't wait for the moment we won't have to maintain this cruft. Sooooon.